libs/networking/http_mock_client/Http_mock_client.ml

```diff
@@ -105,12 +105,16 @@ let body_of_file ?(trim = false) path =
 let check_body expected_body actual_body =
   let%lwt actual_body_content = Cohttp_lwt.Body.to_string actual_body in
   let%lwt expected_body_content = Cohttp_lwt.Body.to_string expected_body in
-  Alcotest.(check string) "body" expected_body_content actual_body_content;
+  (* Passing "" for the name of the check prevents an otherwise
```

Comment:
would probably have been better to define a wrapper my_alcotest_check_string()
that takes just 2 arguments and call Alcotest with the empty string, so we have
just one place with this comment instead of 4.